packages/studio-ui/src/web/components/Layout/BottomPanel/Debugger/views/Summary.tsx

```diff
@@ -42,7 +42,7 @@ export default class Summary extends React.Component<Props> {
         <Dialog
           suggestions={this.props.event.suggestions}
           decision={this.props.event.decision}
-          stacktrace={this.props.event.state?.__stacktrace || []}
+          stacktrace={this.props.event?.state?.__stacktrace || []}
```

Comment:
Why make event optional here, while all other properties around it expect event to be defined ? Doesn't seem useful